src/helpers.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
+_DUPLICATED_TEST_FUNCTIONS_FOUND_ERROR_CODE=100
```

Comment:
From which number were the codes free, could it have been from 128? @khru ?

Reply:
I couldn't make it using this approach of exit codes. I changed it to use a global flag `_DUPLICATED_TEST_FUNCTIONS_FOUND`